code/game/machinery/portable_turret.dm

```diff
@@ -448,7 +448,9 @@
 
 	if(check_anomalies)	//if it's set to check for xenos/simpleanimals
 		for(var/mob/living/simple_animal/SA in turretview)
-			if(!SA.stat && (!SA.has_unlimited_silicon_privilege || !(faction in SA.faction)) ) //don't target dead animals or NT maint drones.
+			if(isbot(SA))
+				continue
+			if(!SA.stat && (!SA.has_unlimited_silicon_privilege || !(faction in SA.faction))) //don't target dead animals or NT maint drones.

```

Comment:
Otherwise beepsky/secbots/mulebots would be able to open all doors on their path.